Makefile

```diff
@@ -78,20 +78,24 @@ build-ui: yarn-install lint-ui ## Build packages
 build-server: lint-server ## Build backend
 	python setup.py bdist_wheel
 
-install-server: ## Install backend
-	pip install --upgrade dist/elyra-*-py3-none-any.whl
+build: build-server build-ui
 
-install-external-extensions:
-	pip install --upgrade jupyterlab-git==0.20.0
-	jupyter labextension install --no-build @jupyterlab/toc@4.0.0
+install-server: build-server ## Install backend
+	pip install --upgrade dist/elyra-*-py3-none-any.whl
 
-install: build-ui build-server install-server install-external-extensions ## Build and install
+install-ui: build-ui
 	$(call LINK_LAB_EXTENSION,application)
 	$(call LINK_LAB_EXTENSION,ui-components)
 	$(call INSTALL_LAB_EXTENSION,theme)
 	$(call INSTALL_LAB_EXTENSION,code-snippet)
 	$(call INSTALL_LAB_EXTENSION,pipeline-editor)
 	$(call INSTALL_LAB_EXTENSION,python-runner)
+
+install-external-extensions:
+	pip install --upgrade jupyterlab-git==0.20.0
+	jupyter labextension install --no-build @jupyterlab/toc@4.0.0
```

Comment:
Should we create macros for the git and toc versions that are placed at the top of the file?
```make
GIT_VERSION:=0.20.0
TOC_VERSION:=4.0.0

pip install --upgrade jupyterlab-git==$(GIT_VERSION)
jupyter labextension install --no-build @jupyterlab/toc@$(TOC_VERSION)
```

Reply:
Looks good - just confirmed this portion of the build passes - thanks for the update.